source/_posts/2015-09-13-home-assistant-meets-ifttt.markdown

```diff
@@ -61,14 +61,17 @@ Setup Maker channel with event name `HA_FIRE_ALARM` and Twitter channel to tweet
 ```yaml
 # Configuration.yaml entry
 automation:
-  alias: Post a tweet when fire alarm is triggered
+- alias: Post a tweet when fire alarm is triggered
+  trigger:
+   - platform: mqtt
+     mqtt_topic: home/alarm/fire
+     mqtt_payload: 'on'
 
-  platform: mqtt
-  mqtt_topic: home/alarm/fire
-  mqtt_payload: 'on'
+  condition: use_trigger_values
```

Comment:
Let's just remove it as it is a wrong example which might confuse users who are trying to setup other automation. I consider `use_trigger_values` an advanced setting that I do not want to expose ot new users.